subprojects/docs/src/docs/release/notes.md

```diff
@@ -38,6 +38,10 @@ Please refer to the [userguide](userguide/dependency_verification.html) to figur
 We would like to give special thanks to [Vladimir Sitnikov](https://github.com/vlsi) for his feedback and inspiration.
 A lot of the work on this feature is, in particular, available to previous versions of Gradle via his [Checksum Dependency Plugin](https://github.com/vlsi/vlsi-release-plugins/tree/master/plugins/checksum-dependency-plugin).
 
+## Gradle source code for plugin developers in Eclipse
+
+Plugin authors using Eclipse will now have sources of `gradleApi()`, `gradleTestKit()` and `localGroovy()` dependencies downloaded and attached for navigation in the IDE.
```

Comment:
Is there a JetBrains issue we could link to that explains why this doesn't work in intellij yet? or at least explain what the behavior in Intellij is for now?

Reply:
I couldn't find one. So far I mentioned the issue in Slack so JB team is aware. I will create an issue in IDEA issue tracker and reference it here. And explain the current behaviour.

Reply:
Thanks!